client-side/src/components/settings/constantSetting.js

```diff
@@ -0,0 +1,2 @@
+export const SEND_PREFERENCE='send preference';
+export const CHANGE_RINGTONE='change ringtone';
```

Comment:
We change this component and his location in the project
do one export like this:  export default CONSTANTS;